universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -17,6 +17,8 @@
   /**
    * Get the {@link Block}s this applicant needs to fill out or has filled out for this program.
    *
+   * <p>All blocks where {@link Block#isEnumerator()} are <b>always</b> in-progress.
```

Comment:
question: why is this the case? Couldn't we just check whether they've seen this question before, and then show them that question in the review page to make sure they don't need to change anything?

Reply:
This was a UX thing I thought we were doing... but maybe I just assumed it was.

Reply:
I don't recall if/what the UX decision was for this. I could see either way...

Reply:
ended up getting rid of this. not hard to add it back later.